Microsoft.Toolkit.Uwp.UI.Controls/FlexPanel/Enums/FlexAlignSelf.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See the LICENSE file in the project root
+// for the license information.
+//
+// Author(s):
+//  - Laurent Sansonetti (native Xamarin flex https://github.com/xamarin/flex)
+//  - Stephane Delcroix (.NET port)
+//  - Ben Askren (UWP/Uno port)
+//
+using System;
+using System.ComponentModel;
+using System.Globalization;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Values for <see cref="P:Microsoft.Toolkit.Uwp.UI.Controls.FlexItem.AlignSelf" />.
+    /// </summary>
+    [TypeConverter(typeof(FlexAlignSelfTypeConverter))]
+    public enum FlexAlignSelf
```

Comment:
Rename to FlexAlignment. Maybe this enum should be both this and FlexItemAlignment and remove Auto